src/EnergyPlus/LowTempRadiantSystem.hh

```diff
@@ -145,25 +150,33 @@ namespace LowTempRadiantSystem {
         Array1D<Real64> SurfaceFrac;     // Fraction of flow/pipe length or electric power for a particular surface
         Real64 TotalSurfaceArea;         // Total surface area for all surfaces that are part of this radiant system
         LowTempRadiantControlTypes ControlType; // Control type for the system (MAT, MRT, Op temp, ODB, OWB)
+        LowTempRadiantSetpointTypes SetpointType;   // Setpoint type for the syste, (HalfFlowPower or ZeroFlowPower)
         Real64 HeatPower;             // heating sent to panel in Watts
         Real64 HeatEnergy;            // heating sent to panel in Joules
 
         // Default Constructor
-            RadiantSystemBaseData()
-                : SchedPtr(0), ZonePtr(0), NumOfSurfaces(0), TotalSurfaceArea(0.0), ControlType(LowTempRadiantControlTypes::MATControl)
-            {
-            }
+        RadiantSystemBaseData()
+            : SchedPtr(0), ZonePtr(0), NumOfSurfaces(0), TotalSurfaceArea(0.0), ControlType(LowTempRadiantControlTypes::MATControl),
+              SetpointType(LowTempRadiantSetpointTypes::halfFlowPower)
+        {
+        }
 
         LowTempRadiantControlTypes processRadiantSystemControlInput(std::string const& controlInput,
                                                                          std::string const& controlInputField
         );
-
-        Real64 setRadiantSystemControlTemperature();
         
+        LowTempRadiantSetpointTypes processRadiantSystemSetpointInput(std::string const& controlInput,
+                                                                      std::string const& controlInputField
+        );
+        
+        Real64 setRadiantSystemControlTemperature();
+
         Real64 calculateOperationalFraction(Real64 const offTemperature, Real64 const controlTemperature, Real64 const throttlingRange);
         
+        Real64 setOffTemperatureLowTemperatureRadiantSystem(int const scheduleIndex, Real64 const throttlingRange);
+        
         virtual void calculateLowTemperatureRadiantSystem(ZoneTempPredictorCorrectorData &dataZoneTempPredictorCorrector, Real64 &LoadMet) = 0;
-                
+                        
```

Comment:
Did you catch that this blank line is now filled with spaces?

Reply:
@rraustad Whoops--I had missed that.  Nice catch--thanks!  Just fixed it and made another commit.